recipes/radioactivedecay/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "radioactivedecay" %}
+{% set version = "0.3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radioactivedecay-{{ version }}.tar.gz
+  sha256: ddb0a1054cf659db465a7355be3f8ac203e81206aa90b07798ac0eeafb65d99a
+
+build:
+  number: 0
+  skip: true   # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib_resources  # [py<37]
+    - matplotlib-base
+    - networkx
+    - numpy
+    - python
+    - scipy
+    - sympy
+
+test:
+  imports:
+    - radioactivedecay
+  commands:
+    - pip check
+    - python -m unittest discover
+  requires:
+    - pip
+    - python
+  source_files:
+    - tests/
+
+about:
+  home: https://github.com/alexmalins/radioactivedecay
+  license: MIT
```

Probably should also note the data file license:
```suggestion
  license:
    - MIT
    - LicenseRef-ICRP-07
```

---

Done thanks. conda-build did not like the list formatting, but worked with multiple `license:` and `license_family:` entries.

Thanks for the review and the suggestions, @dopplershift !

---

You need to use an "spdx license expression" (basically `XXX OR YYY` or `XXX AND YYY`, I guess `AND` applies here?)

---

You can then drop `license_family` now as it's automatically generated for valid SPDX expressions in `license`

---

Fixed, thanks @chrisburr